browser/renderer_context_menu/test/render_view_context_menu_browsertest.cc

```diff
@@ -0,0 +1,93 @@
+#include "brave/app/brave_command_ids.h"
+#include "brave/browser/ui/browser_commands.h"
+#include "brave/browser/ui/tabs/brave_tab_strip_model.h"
+#include "brave/browser/ui/tabs/features.h"
+#include "brave/browser/ui/tabs/split_view_browser_data.h"
+#include "chrome/browser/renderer_context_menu/render_view_context_menu_browsertest_util.h"
+#include "chrome/browser/renderer_context_menu/render_view_context_menu_test_util.h"
+#include "chrome/browser/ui/browser.h"
+#include "chrome/test/base/in_process_browser_test.h"
+#include "content/public/browser/web_contents.h"
+#include "content/public/test/browser_test.h"
+#include "content/public/test/browser_test_utils.h"
+#include "third_party/blink/public/mojom/context_menu/context_menu.mojom.h"
+
+using namespace content;
```

Comment:
I don't think using namespace is recommended in Chromium.

Reply:
Yes, Removed it.